packages/wallet-adapter-core/src/WalletCore.ts

```diff
@@ -194,15 +196,28 @@ export class WalletCore extends EventEmitter<WalletCoreEvents> {
     });
   }
 
-  private async setWallets(wallets: readonly AptosWallet[]) {
+  /**
+   * Set potential Standard compatible SDK + extension wallets
+   *
+   * Loop over local SDK and Extensions wallets
+   * 1) check it is Standard compatible
+   * 2) Update their readyState to Installed (for a future UI detection)
+   * 3) push the wallet into a local wallets array
+   * 4) standardize each wallet to the Wallet Plugin type interface for legacy compatibility
+   *
+   * @param extensionwWallets
+   */
+  private setWallets(extensionwWallets: readonly AptosWallet[]) {
```

Comment:
@hardsetting please take another look, I have updated the code.

Can see a live example here https://aptos-labs.github.io/aptos-wallet-adapter/nextjs-example-testing/

Reply:
Just checking that `this.fetchAptosWallets()` is always called so that `this._aptos_wallets` is always initialized

Reply:
yes, it is part of the `WalletCore` constructor method, so always called
https://github.com/aptos-labs/aptos-wallet-adapter/blob/main/packages/wallet-adapter-core/src/WalletCore.ts#L132